rollup.config.js

```diff
@@ -169,23 +174,28 @@ export default [
     ]
   }, {
     ...config,
-    plugins: makePlugins(true),
+    plugins: makePlugins(true, 'ADVANCED_OPTIMIZATIONS'),
     onwarn: () => {},
     output: [
       {
         sourcemap: false,
         legacy: true,
         freeze: false,
-        file: './dist/urql.min.js',
-        format: 'cjs'
-      },
+        file: './dist/urql.es.min.js',
+        format: 'esm'
+      }
+    ]
+  }, {
+    ...config,
+    plugins: makePlugins(true, 'SIMPLE_OPTIMIZATIONS'),
```

Comment:
I suppose the problem is that people won’t use any of our heavily optimised mini field bundles by default, which negates some of the benefits 😔

Reply:
I can split it up in 4 so that we can do advanced optimizations on the normal es bundle as well